src/sentry/incidents/subscription_processor.py

```diff
@@ -562,6 +622,106 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         # before the next one then we might alert twice.
         self.update_alert_rule_stats()
 
+    def get_anomaly_detected(
+        self, anomaly
```

Comment:
Are we passing the `anomaly` here?

Or is it just the results from seer?
shoudl this be `seer_results` or something?

Reply:
We pass each anomaly object here.